sentry-rails/Gemfile

```diff
@@ -24,6 +24,7 @@ if rails_version > Gem::Version.new("7.0.0")
   gem "rails", github: "rails/rails"
 else
   gem "rails", "~> #{rails_version}"
+  gem "psych", "~> 3.0.0"
```

Comment:
Because Rails 6 doesn't require `psych`'s version, it could still be bumped to a incompatible version. So we need to lock it here.